kolibri/plugins/user/assets/src/views/SignInPage.vue

```diff
@@ -369,6 +366,13 @@
         this.setSuggestionTerm(newVal);
       },
     },
+    created() {
+      // Only get facilities that meet our criteria for listing the users
+      const facilityIdsToFetch = this.facilities
+        .filter(f => f.num_users <= MAX_USERS_FOR_LISTING_VIEW)
+        .map(f => f.id);
+      this.$store.dispatch('signIn/fetchUsersForFacilities', facilityIdsToFetch);
```

Comment:
What happens if this is an empty list? Should either check for that here and not dispatch the action, or inside the action just instantly resolve without the request.